src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdNewCaseCreator.java

```diff
@@ -206,9 +217,8 @@ private long createNewCaseInCcd(
         }
     }
 
-    @SuppressWarnings("unchecked")
-    private Map<String, Object> caseDataWithExceptionRecordId(Object caseData, String exceptionRecordId) {
-        Map<String, Object> data = new HashMap((Map<String, Object>) caseData);
+    private Map<String, Object> caseDataWithExceptionRecordId(Map<String, Object> caseData, String exceptionRecordId) {
+        Map<String, Object> data = new HashMap<>(caseData);
```

Comment:
do we need to convert this again to new map ? Can we not use caseData as it is already a map ?

Reply:
iirc, this was so to not use original object as it conflicted with end result. i'm sure this can be tracked backwards... and here it is: #785